### PR TITLE
Add global readme guidance to run in local development

### PR DESCRIPTION
**Pull request by lloistborn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,36 @@
...
+### Running in Local Machine
+1. **Make sure gatsby is installed as global command** (first time only)
+```
+// install as global command
+> npm install -g gatsby-cli
+
+// verify gatsby-cli installed by checking the version
+> gatsby --version
+```
+2. **Go to root project and install all dependencies** (first time only)
```
Comment (ariefrahmansyah) [inline README.md:18]: Should we add the guidance to clone the repository as well?

File: README.md [hunk 2]
```diff
@@ -1,99 +1,36 @@
...
+```
+// change directory to root project web-frontend folder
+> cd web-frontend
+
+// install all the dependencies
+/web-frontend > npm install .
+```
+3. **While still in root project build and run the apps**
```
Comment (ariefrahmansyah) [inline README.md:26]: App, not apps. It's singular.

File: README.md [hunk 3]
```diff
@@ -1,99 +1,36 @@
...
+```
+// build and run the apps
```
Comment (ariefrahmansyah) [inline README.md:28]: App, not apps. It's singular.

File: README.md (hunk 1 above)
Comment (lloistborn) [reply README.md:18]: good point i will add

File: README.md (hunk 2 above)
Comment (lloistborn) [reply README.md:26]: nice catch

File: README.md [hunk 4]
```diff
@@ -1,99 +1,36 @@
...
+// verify gatsby-cli installed by checking the version
+> gatsby --version
+```
+2. **Go to root project and install all dependencies** (first time only)
+```
+// change directory to root project web-frontend folder
+> cd web-frontend
+
+// install all the dependencies
+/web-frontend > npm install .
```
Comment (ariefrahmansyah) [inline README.md:24]: Should just write `> npm install .`

The prefix `/web-frontend` can be misleading.

Comment (lloistborn) [reply README.md]: solid point, will change into `> npm install . `

**Pull request merged**